config/operator.yaml

```diff
@@ -19,6 +19,8 @@ metadata:
   namespace: default
   labels:
     operator.knative.dev/release: devel
+    app.kubernetes.io/version: devel
```

Comment:
Can you add `app.kubernetes.io/name: knative-operator` as well? The `name` key is added to Deployments/STSs/DSs and Services.

Also, add the new labels to `spec.template.metadata.labels`.

Can you append this Service too?

```
---
apiVersion: v1
kind: Service
metadata:
  labels:
    serving.knative.dev/release: devel
    app.kubernetes.io/name: knative-operator
    app.kubernetes.io/version: devel
    app.kubernetes.io/part-of: knative-operator
  name: knative-operator
  namespace: knative-operator
spec:
  ports:
  # Define metrics for it to be accessible within service meshes.
  - name: http-metrics
    port: 9090
    targetPort: 9090
  selector:
    name: knative-operator
```

Also, why is Istio injection disabled on this deployment?

Reply:
I do not know why istio is disabled. it has been there since the beginning.
Why do we need to add this service?

Reply:
It is to create a servicemonitor targeting the operator pods

Reply:
Let's open another issue about the connection to monitoring.

Reply:
Ok